src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -132,7 +132,17 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
             <div className="profile-extras">
               {renderedProfileQuestionAnswers}
             </div>
+            {/* Proper styling + add editing possibility */}
```

Comment:
> Proper styling + add editing possibility

Reply:
`UserProfileModal` should presumably always just be static content?

`EditProfileForm` appears to be the component used for changing party name/profile picture/etc. That's where it would make sense for all of this editing to happen.

Reply:
Implemented proper styling in https://github.com/sparkletown/sparkle/pull/1667/commits/3da671cd41b87e2e89594ea2e62bcb2fb7fde73d

Reply:
![image](https://user-images.githubusercontent.com/63194656/123612420-2f9d2600-d80b-11eb-9a1c-f9f59bb7175d.png)

Reply:
Will add the editing, but it should not be a blocker

Reply:
Looks good. Sounds good.